pkg/apis/storage/types.go

```diff
@@ -80,3 +91,18 @@ type StorageClassList struct {
 	// Items is the list of StorageClasses
 	Items []StorageClass
 }
+
+// VolumeBindingMode indicates how PersistentVolumeClaims should be bound.
+type VolumeBindingMode string
+
+const (
+	// VolumeBindingImmediate indicates that PersistentVolumeClaims should be
+	// immediately provisioned and bound.
+	VolumeBindingImmediate VolumeBindingMode = "Immediate"
+
+	// VolumeBindingTopologyAware indicates that PersistentVolumeClaims should not
+	// be provisioned and bound until a Pod is created that references the
+	// PeristentVolumeClaim.  The volume provisioning and binding will occur during
+	// Pod scheduing.
+	VolumeBindingTopologyAware VolumeBindingMode = "TopologyAware"
```

Comment:
As a constant, this doesn't really explain what it will do.  What's wrong with "FirstConsumer" or something ?

Reply:
"FirstPod"?  "WaitForPod"?

Reply:
How about "WaitForFirstConsumer"?  I like how the "wait" contrasts with "immediate"